src/lib/__tests__/AnnotationControlsFSM-test.js

```diff
@@ -0,0 +1,90 @@
+import AnnotationControlsFSM, { AnnotationInput, AnnotationState } from '../AnnotationControlsFSM';
+import { AnnotationMode } from '../AnnotationControls';
+
+let annotationControlsFSM;
+
+describe('lib/AnnotationControlsFSM', () => {
+    beforeEach(() => {
+        annotationControlsFSM = new AnnotationControlsFSM();
+    });
+
+    [
+        {
+            state: AnnotationState.NONE,
+            input: AnnotationInput.CLICK,
+            mode: AnnotationMode.HIGHLIGHT,
+            nextState: AnnotationState.HIGHLIGHT,
```

Comment:
Does `nextState` and `output` ever differ?

Reply:
`nextState` could be `highlight_temp` and output is `highlight`